### PR TITLE
Join SR2 Slingshot  and need review

### PR DESCRIPTION
**Pull request by coinmini**
Join SR2 Slingshot  and need review 
请审核

### CODE REVIEW

File: participants/Panfil.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Our application is mainly upload and search based. It will be a Filcoin user admin panel. Users can use it easily to upload and retrieve kinds of files in the future.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+No using curated dataset.
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+We provide the data from our IDC switch log Now.
```
Comment (dineshshenoy) [inline participants/Panfil.md:28]: Can you explain why the IDC switch log data is useful for any users?  What can they do with that data?

Comment (coinmini) [reply participants/Panfil.md:28]: Actually, I have no idea about the value of the  IDC switch log. However, the IDC data center boss want to use the Filecoin store service

Comment (dineshshenoy) [reply participants/Panfil.md:28]: To be eligible for the competition, it needs to be useful to others in some way.  Please check and let us know!

Comment (coinmini) [reply participants/Panfil.md:28]: hi, I asked  the IDC boss, they reply to me about the use just now:  they need huge data store for the IDC switch data store for some reason here: the log review, safety analyse, active defense strategy. The idc switch logs are kind of cold data.  Others can also use the huge data for big data analyse.
I just think: I use the filecoin to supply the place to store. The IDC datacenter need place to store their data. Thats what the market  is. Every customer have their own reason. Seems github cannot upload picture. I will post the talk between the idc boos and me at the slack. https://files.slack.com/files-pri/TEHTVS1L6-F01CXDM9W4T/idc.jpeg

**Review by dineshshenoy — APPROVED** [review]

**Pull request merged**